pkg/build/controller/controller.go

```diff
@@ -164,6 +159,7 @@ func (bc *BuildController) nextBuildPhase(build *buildapi.Build) error {
 
 	if _, err := bc.PodManager.CreatePod(build.Namespace, podSpec); err != nil {
 		if errors.IsAlreadyExists(err) {
+			bc.Recorder.Eventf(build, "failedCreate", "Pod already exists: %s/%s", podSpec.Namespace, podSpec.Name)
```

Comment:
yup, we don't want to update the build because the pod existing may indicate another controller handling the build